src/Silhouette.js

```diff
@@ -67,12 +88,57 @@ class Silhouette {
         const imageData = ctx.getImageData(0, 0, width, height);
 
         this._data = new Uint8ClampedArray(imageData.data.length / 4);
+        this._colorData = imageData.data;
 
         for (let i = 0; i < imageData.data.length; i += 4) {
             this._data[i / 4] = imageData.data[i + 3];
         }
     }
 
+    colorAtNearest (vec, dst) {
+        dst = dst || new Uint8ClampedArray(4);
+        if (!this._colorData) {
```

Comment:
I wonder if it would be faster to define two versions of this function. One that returns `dst.fill` and one that returns `getColor4b`. Set `colorAtNearest` to the right implementation when initializing `Silhouette` and updating it.